apps/routes-api/src/app/controllers/routes.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Request, Response } from 'express';
+import { RoutesService }  from '../services/routes.service';
+
+class RoutesController {
+
+    private routesService:  RoutesService
+
+    constructor(){
+        this.routesService = new RoutesService();
+    }
+
+    public removeRoute = async (req: Request, res: Response, next: NextFunction) => {
+      try {
+        const  id  = req.params.id
+        this.routesService.removeRoute(id)
+        res.sendStatus(200)
+      } catch (error) {
+        next(error);
+      }
+    }
+}
+export default RoutesController;
```

Comment:
Could you please add a newline in each file that doesn't have it at the end? If not, git will complain about them 🙇🏼 Thanks!